deps/verifier/src/snp/mod.rs

```diff
@@ -104,7 +108,8 @@ impl Verifier for Snp {
 
         verify_report_signature(&report, &cert_chain, &self.vendor_certs)?;
 
-        if report.version != 2 {
+        // See Trustee Issue#589 https://github.com/confidential-containers/trustee/issues/589
+        if report.version >= REPORT_VERSION_MIN_SUPPORT && report.version <= REPORT_VERSION_MAX_SUPPORT {
```

Comment:
I think you mean `if version < MIN || version > MAX `

Reply:
Oops, my bad. Thanks!